homeassistant/components/nut/__init__.py

```diff
@@ -234,15 +233,15 @@ def _get_alias(self) -> str | None:
         self.ups_list = ups_list
         return list(ups_list)[0]
 
-    def _get_device_info(self) -> DeviceInfo | None:
+    def _get_device_info(self) -> dict[str, str] | None:
         """Get the ups device info from NUT."""
         if not self._status:
             return None
 
         manufacturer = _manufacturer_from_status(self._status)
         model = _model_from_status(self._status)
         firmware = _firmware_from_status(self._status)
-        device_info = DeviceInfo()
+        device_info: dict[str, str] = {}
         if model:
```

Comment:
We are still mixing the concept of Home Assistant DeviceInfo and NUT's device info.

I suggest making this return a `dataclass` called `NUTDeviceInfo` instead, then its really clear on what type of data we are returning

Reply:
Done, let me know if looks good to you.